src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -125,6 +106,69 @@ export const TravelAidScreenComponent = ({
   );
 };
 
+const TimeInfo = ({state}: {state: FocusedEstimatedCallState}) => {
+  const {language, t} = useTranslation();
+  const styles = useStyles();
+  const {themeName} = useTheme();
+  const {focusedEstimatedCall, status} = state;
+
+  const clock = formatToClock(
+    focusedEstimatedCall.aimedDepartureTime,
+    language,
+    'round',
+  );
+
+  switch (status) {
+    case TravelAidStatus.EndOfLine:
+      return null;
+    case TravelAidStatus.NoRealtime:
+    case TravelAidStatus.NotGettingUpdates:
+      return (
+        <ThemeText type="body__secondary--bold">
+          {t(TravelAidTexts.clock(clock))}
+        </ThemeText>
+      );
+    default:
```

Comment:
Instead of `default:` my preference is having

```ts
case TravelAidStatus.NotYetArrived:
case TravelAidStatus.Arrived:
case TravelAidStatus.BetweenStops:
``` 

… so if a new status is added the developer gets a heads up that they should decide what time info that new status should give.